### PR TITLE
shortlist page

### PR DESCRIPTION
**Pull request by ralph-hawkins**
changed content. 

- if you have more than 1 response the page tells you to follow the guidance on how to shortlist.

- if you don't have any responses the page tells you to start again.

also added back link to the overview page.

### CODE REVIEW

**Comment by TheDoubleK** [comment]
Question: this has changed things so that the CSV download link is always shown.  Is this right?  I thought we had agreed that responses should only be downloadable once the brief is closed.

**Comment by TheDoubleK** [comment]
Should we also tell buyers with zero eligible responses that "Suppliers that did not meet all your essential requirements have already been told they were unsuccessful."?

**Comment by TheDoubleK** [comment]
I've fixed some of the logic about when text should be shown, and made the tests pass, but earlier questions are still relevant.  Changes need to be made either way depending on whether we want to allow csv download on open briefs.

**Comment by TheDoubleK** [comment]
Updated, think this is all good now.

New screenshots:
1. Brief is closed
screen shot 2016-04-14 at 13 34 15

2. Brief is still open
screen shot 2016-04-14 at 13 35 40

**Comment by ralph-hawkins** [comment]
@TheDoubleK you’re right, we should keep the line about 'Suppliers that did not meet all your essential requirements have already been told they were unsuccessful.' if there are 0 responses.

**Comment by TheDoubleK** [comment]
Latest view of zero-eligible version of the page:

screen shot 2016-04-14 at 13 47 25

**Comment by superroz** [comment]
content for zero-eligible should be:

No suppliers met your essential skills and experience requirements.

If you still need this service, you should start the buying process again. Consider:
- talking to suppliers before you start
- rewriting your requirements

Back to overview

**Comment by TheDoubleK** [comment]
Let's sort out the conflicting versions in person and come up with some "final" words.

**Comment by superroz** [comment]
No suppliers met your essential skills and experience requirements. 

All the suppliers who applied have already been told they were unsuccessful.

If you still need this service, you should start the buying process again. Consider:

- talking to suppliers before you start
- rewriting your requirements

Back to overview

File: app/templates/buyers/brief_responses.html [hunk 1]
```diff
@@ -12,20 +12,22 @@
...
 {% endblock %}
 
-{% set responses_count = response_counts['failed'] + response_counts['eligible'] %}
-
 {% block main_content %}
 <div class="grid-row">
     <div class="column-two-thirds">
         {% with
-        heading = ("Responses to &lsquo;" + brief_data.get('title', brief_data['lotName']) + "&rsquo;")|safe,
+        heading = ("Shortlist suppliers")|safe,
```
Comment (tombye) [inline app/templates/buyers/brief_responses.html]: This no longer needs the `safe` filter as it doesn't contain any characters that would be escaped.

**Comment by tombye** [comment]
I fixed [my own issue](https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/251#discussion-diff-59891093) but ran it past @TheDoubleK. Everything else is good so :+1:

**Pull request merged**